src/Components/Nav/NavButton.jsx

```diff
@@ -9,6 +9,7 @@ function NavButton({ name, design = "default", link, icon = null }) {
       to={link}
       activeClassName="nav-button-active"
       className="nav-button-wrapper"
+      replace
```

Comment:
why you added replace?

Reply:
I added that to avoid the following warning:
<img width="609" alt="image" src="https://user-images.githubusercontent.com/16488024/112747757-a7af4200-8fb7-11eb-96fb-52b2ceb65af8.png">

But that was not the solution and the warning is just a dev warning. The HashRouter will just do nothing in that situation.

I removed the "replace" prop.